Gems/EMotionFX/Code/Source/Editor/SkeletonModel.cpp

```diff
@@ -627,21 +665,21 @@ namespace EMotionFX
             actor->ExtractBoneList(lodLevel, &boneListPerLodLevel[lodLevel]);
         }
 
-        for (size_t nodeIndex = 0; nodeIndex < numNodes; ++nodeIndex)
+        for (size_t nodeIndex = 1; nodeIndex < numNodes; ++nodeIndex)
```

Comment:
nit: might worth to add a comment in here saying start from 1 because the "character node" is skipped, right?